Homework2/Trie/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using TrieDataStructure;
+
+if (!Trie.Tests())
+{
+    Console.WriteLine("Tests failed.");
+}
```

Comment:
```suggestion
    Console.WriteLine("Tests failed.");
    System.Environment.Exit(1);
}
```
И дальше без else. Это прямой аналог return 1 из Си.